service/src/main/java/tds/session/services/impl/SessionServiceImpl.java

```diff
@@ -23,4 +23,11 @@ public SessionServiceImpl(SessionRepository sessionRepository) {
     public Optional<Session> getSessionById(UUID id) {
         return sessionRepository.getSessionById(id);
     }
+
+    @Override
+    public void pause(final UUID sessionId, final String newStatus) {
+        // TODO:  Throw an exception if attempting to pause a Session that is already paused?
```

Comment:
Wanted to call special attention to this:  Is it exception-worthy to try pausing a session that's already paused?  I tend to think so, but I don't have a strong reason why.  When a session is paused, the `dateChanged` and `dateEnd` fields in the `session.session` table will be updated to reflect when the status change happened.  At this point, I'm not sure what the far-reaching effects of changing those fields are - maybe nothing.  However, I could see the controller returning a 409 (or 412 preconditions failed?) for updating an already paused session